src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.ps1

```diff
@@ -37,14 +37,19 @@ function Test-RecoveryServicesVaultCRUDTests
 		Assert-NotNull($vault.Type)
 	}
```

Comment:
Indent this properly so that there is no confusion. It looks like the test ended here and code below is just hanging around.